src/chapters/introduction/using-this-book.rst

```diff
@@ -1,2 +1,31 @@
 Using This Book
 ===============
+
+Throughout this book, you will find a variety of different sections and practice exercises.
+We are writing this guide to help you make the most of the book.
+
+Concept checks
+--------------
+
+Many sections end with a "Check Your Understanding".
+This section is full of questions for you to double check that you understand the concepts in the reading.
+Your score does not count towards your final grade in the class
+You should be using your score for self-evaluation on concepts you may need help with in-class.
+
+Examples
+--------
+
+Examples are times when we tie a concept we have just learned to a potential real world application of the concept.
```

Comment:
Either remove "of the concept" or rephrase as "of that concept".

Reply:
Thanks!